src/System.ComponentModel.TypeConverter/src/System/ComponentModel/EventHandlerList.cs

```diff
@@ -69,21 +57,25 @@ internal EventHandlerList(Component parent)
         ///    <para>[To be supplied.]</para>
```

Comment:
There's some work to include better documentation, so that will take care of it.